bin/ffigen.dart

```diff
@@ -11,7 +11,7 @@ import 'package:ffigen/src/header_parser.dart' as parser;
 import 'package:logging/logging.dart';
 import 'package:yaml/yaml.dart' as yaml;
 
-var _logger = Logger('ffigen.dart');
+var _logger = Logger('ffigen.ffigen');
```

Comment:
Shouldn't this be `Logger('ffigen')`, such that it becomes the root of the other loggers?

Reply:
This should be the root logger as this is the ffigen.dart script which is just another part of ffigen.
The root logger is created automatically in the Logger constructor when we create a logger with a `.` to separate names.
So root of all loggers we created is Logger('ffigen').

Reply:
👍